### PR TITLE
update README

### PR DESCRIPTION
**Pull request by rhagarty**
- included instructions for installing vector plugin, which includes code lines to insert into config files
- moved installing DSX local plugins to troubleshooting section
- fixed HDP plugin table to include tmdbsimple
- added note about significance of cells with %%spark

### CODE REVIEW

**Comment by dilipbiswal** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -102,17 +104,49 @@ This code pattern was tested with HDP Search (`Solr`) v6.6.2 and assumes that it
...
-    To test out your changes, try out the samples shown in the [Examples](https://github.com/saaay71/solr-vector-scoring#example) section of the plugin's README. This will require you to create test collections - see the [Running Solr Docs](https://lucene.apache.org/solr/guide/6_6/running-solr.html#running-solr) for instructions on how to do that.
+    To help simplify and consolidate the instructions, the important steps are repeated below:
+  
+    > Note: The following instructions assume the installation directory for Solr is `/opt/lucidworks-hdpsearch/solr`.
+  
+    1. Download the [VectorPlugin.jar](https://github.com/saaay71/solr-vector-scoring/blob/master/VectorPlugin.jar) file.
+    
+    2. Create directory `plugins` in the `dist` directory of the Solr installation location.
+    
+    3. Copy VectorPlugin.jar to /opt/lucideworkds-hdpsearch/solr/dist/plugins/
```
Comment (markstur) [inline README.md:123]: backtick around VectorPlugin.jar to format it like youi do other files.  The dir too I suppose.

File: README.md [hunk 2]
```diff
@@ -185,14 +219,16 @@ This code pattern relies upon a few python plugins for everything to work togeth
...
-| tmdbsimple | DSX Node |
+| tmdbsimple | DSX node and all nodes of HDP cluster |
 | IPython  | DSX node |
 | paramiko | All nodes of HDP cluster | 
 | numpy | All nodes of HDP cluster |
 | simplejson | DSX node and all nodes of HDP cluster |
 | urllib2 | DSX node and all nodes of HDP cluster |
 | solrcloudpy | DSX node and all nodes of HDP cluster |
 
+> Note regarding DSX node plugins: The Jupyter notebook running in DSX Desktop will attempt to install each of the required plugins, so no futher action should be needed. If, however, there are issues and a plugin needs to be manually installed, see the [Troubleshooting](#troubleshooting) section below for instructions.
```
Comment (markstur) [inline README.md:230]: DSX "Desktop" vs "Local" could be confusing.

File: README.md [hunk 3]
```diff
@@ -273,8 +289,18 @@ To run the notebook you will need to start DSX Local. Below are the steps that n
...
         * `tmdb.API_KEY`
 
-      > Note: Some variables such as tmdb.API_KEY, SOLR_HOST etc are defined two times in the notebook as they are needed in the DSX node and Livy node. Please make sure to update all the occurrence of the specific variable in the notebook.
+    Some variables such as tmdb.API_KEY, SOLR_HOST etc are defined twice in the notebook as they are needed in the DSX node and the Livy node. 
+    
+    A Livy node refers to notebook cells that access the HDP cluster using the Livy interface. These cells can be distinquished from other notebook cells by the following tag located at the top of the cell:
 
+    Some variables such as tmdb.API_KEY, SOLR_HOST etc are defined twice in the notebook as they are needed in both the DSX node and the HDP cluster nodes. 
+    
+    Note that in the notebook, the HDP cluster nodes are acceesed via the Livy interface. Cells that access the HDP cluster are distinquishable by the following tag located at the top of the cell:
```
Comment (markstur) [inline README.md:298]: typo on "acceesed"

Comment (markstur) [reply README.md:298]: and "distinquishable"

File: README.md [hunk 4]
```diff
@@ -273,8 +289,18 @@ To run the notebook you will need to start DSX Local. Below are the steps that n
         * `SSHPASSWORD`
         * `tmdb.API_KEY`
 
-      > Note: Some variables such as tmdb.API_KEY, SOLR_HOST etc are defined two times in the notebook as they are needed in the DSX node and Livy node. Please make sure to update all the occurrence of the specific variable in the notebook.
+    Some variables such as tmdb.API_KEY, SOLR_HOST etc are defined twice in the notebook as they are needed in the DSX node and the Livy node. 
+    
+    A Livy node refers to notebook cells that access the HDP cluster using the Livy interface. These cells can be distinquished from other notebook cells by the following tag located at the top of the cell:
 
+    Some variables such as tmdb.API_KEY, SOLR_HOST etc are defined twice in the notebook as they are needed in both the DSX node and the HDP cluster nodes.
```
Comment (markstur) [inline README.md:296]: this part seems VERY redundant.  Did you forget to delete a couple of paragraphs here?

File: README.md [hunk 5]
```diff
@@ -366,7 +392,27 @@ The example output in the [data/examples](data/examples) folder shows the output
...
   > Solution: You will need to access [The Movie Database API](https://www.themoviedb.org/documentation/api) and follow theses [instructions](https://developers.themoviedb.org/3/getting-started) to get an API key. Then copy the key into the `tmdb.API_KEY = 'YOUR_API_KEY'` line in the notebook. Once you have done that, execute that cell to test your access to TMDb API.
-  
+
+* Error: `Notebook running in DSX Desktop is unable to install python packages or plugins`
+
+  Occasionally, this can happen due to the presence of multiple python environments in the docker container that runs the notebook. If this occurs, we can install the plugins manually by performing the following steps:
+
+  1. Find out the python environment that is used by the notebook by inserting a cell at the top of the notebook. Enter and run the following commands:
+    ```
+    !!which python
```
Comment (markstur) [inline README.md:402]: why !! instead of just !

**Review by markstur — COMMENTED** [review]
See comments

**Pull request merged**